src/main/yamcs/etc/logging.properties

```diff
@@ -0,0 +1,12 @@
+handlers = java.util.logging.ConsoleHandler, java.util.logging.FileHandler
+
+java.util.logging.ConsoleHandler.level = INFO
+
+
+java.util.logging.FileHandler.level = ALL
+java.util.logging.FileHandler.formatter = org.yamcs.logging.CompactFormatter
+java.util.logging.FileHandler.pattern = /opt/yamcs/log/yamcs-server.log
```

Comment:
Should not assume Yamcs has permission to write to `/opt/`.

Reply:
There's no elegant way to make it write to a permission-safe directory without running yamcs as root or running a script as root temporarily in order to make and change the owners of a dir.

Right now, this does not crash yamcs, it just stops the logging it it happens to not be installed in `/opt`.

Once again, right now is not polish time and future proofing, this is a minimum viable product.

We can spend time later, looking at more elegant ways to ensure the log directory exists (probably via maven somehow) but this isn't a road blocker.